[2025-05-21T14:06:16.104+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-21T14:06:16.105+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model'
[2025-05-21T14:06:16.129+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-21T14:06:16.137+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model'
[2025-05-21T14:06:16.171+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-21T14:06:06.292766+00:00 [queued]>
[2025-05-21T14:06:16.178+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-21T14:06:06.292766+00:00 [queued]>
[2025-05-21T14:06:16.179+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-21T14:06:16.190+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): populate_dimensional_model> on 2025-05-21 14:06:06.292766+00:00
[2025-05-21T14:06:16.195+0000] {standard_task_runner.py:60} INFO - Started process 224 to run task
[2025-05-21T14:06:16.197+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'populate_dimensional_model', 'manual__2025-05-21T14:06:06.292766+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmp5kj2ugxk']
[2025-05-21T14:06:16.199+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask populate_dimensional_model
[2025-05-21T14:06:16.244+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509
[2025-05-21T14:06:16.244+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T14:06:06.292766+00:00/task_id=populate_dimensional_model'
[2025-05-21T14:06:16.245+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.populate_dimensional_model manual__2025-05-21T14:06:06.292766+00:00 [running]> on host 3ce5c155e4d2
[2025-05-21T14:06:16.432+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='populate_dimensional_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T14:06:06.292766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T14:06:06.292766+00:00'
[2025-05-21T14:12:25.637+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-21T14:12:25.646+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=populate_dimensional_model, execution_date=20250521T140606, start_date=20250521T140616, end_date=20250521T141225
[2025-05-21T14:12:25.671+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-21T14:12:25.684+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
